code/modules/shuttle/cas_plane.dm

```diff
@@ -314,7 +314,7 @@
 		return
 	var/area/A = get_area(target)
 	if(A.ceiling >= CEILING_DEEP_UNDERGROUND)
-		to_chat(source, span_warning("That target is too deep underground!"))
+		to_chat(source, span_warning("Our payload won't reach this target!"))
```

Comment:
this is lazy make it use an area flag instead of just changing the warning message